### ISSUE TITLE
Sqlite PrimaryKey missing

### ISSUE THREAD

**Issue by powof2** [jchristn/RestDb#6]
Hi,
I tried to create a table following your tutorial (using sqlite), but it looks the "PrimaryKey" is missing?

this is the data to create table (as in your tutorial)
```json
{
  "Name": "person",
  "PrimaryKey": "person_id",  // PrimaryKey is here <------------
  "Columns": [
    {
      "Name": "person_id",
      "Type": "int",
      "Nullable": false
    },
    {
      "Name": "first_name",
      "Type": "nvarchar",
      "MaxLength": 32,
      "Nullable": false
    },
    {
      "Name": "last_name",
      "Type": "nvarchar",
      "MaxLength": 32,
      "Nullable": true
    },
    {
      "Name": "age",
      "Type": "int",
      "Nullable": false
    },
    {
      "Name": "created",
      "Type": "datetime",
      "Nullable": true
    }
  ]
}
```
but the created table has no 'PrimaryKey' information, this is what i got use `http://localhost:8000/restlib/person?_describe`
```json
// no primary key info, they are all false
{
  "Name": "person",
  "Columns": [
    {
      "Name": "person_id",
      "PrimaryKey": false,
      "Type": "Int",
      "Nullable": false
    },
    {
      "Name": "first_name",
      "PrimaryKey": false,
      "Type": "Varchar",
      "Nullable": false
    },
    {
      "Name": "last_name",
      "PrimaryKey": false,
      "Type": "Varchar",
      "Nullable": true
    },
    {
      "Name": "age",
      "PrimaryKey": false,
      "Type": "Int",
      "Nullable": false
    },
    {
      "Name": "created",
      "PrimaryKey": false,
      "Type": "Varchar",
      "Nullable": true
    }
  ]
}
```
I looked into the source and found this line (PostTableCreate.cs, line 38):
```csharp
    Table table = SerializationHelper.DeserializeJson<Table>(md.Http.Request.DataAsString);
    db.CreateTable(table.Name, table.Columns);
```
it looks the table was created with name and columns, but columns have no 'PrimaryKey' info (as you can see in the json data above from your tutorial).

I am new to database/programming, so i don't know what's the right thing to do here. But if this problem is not solved, errors will occur when obtaining user data by person_id later.

Please help, thank you.

**Comment by jchristn** [comment]
Hi, in your request body, add `"PrimaryKey": true,` to the column object as well.

**Comment by jchristn** [comment]
Also I have just published v2.0.6 which will automatically set this for you.  I also added Docker support.  Please let me know if you continue to have any issues!

**Comment by powof2** [comment]
V2.0.6 works as expected, and i don't need to change anything on client side, thank you.
